errors.js

```diff
@@ -0,0 +1,19 @@
+exports.handleCustomErrors = (err, req, res, next) => {
+    if (err) {
+        res.status(err.status).send({ msg: err.msg })
+    } else {
+        next(err);
+    }
+}
+
+exports.handlePsqlErrors = (err, req, res, next) => {
+    console.log(err)
+    if (err.code === "22P02") { // invalid text representation
+        res.status(400).send({ msg: "Bad request" });
+    }
+    else if (err.code === "23502") { // violates not null
+        res.status(400).send({ msg: "Bad request" });
+    } else {
+        next(err);
+    }
```

Comment:
I'm sure these will be important in the future, but at the moment you haven't actually got any tests that are relevant to these error handlers. Only write the code in once you know it will be used!